django_extensions/compat.py

```diff
@@ -145,19 +145,19 @@ def get_model_compat(app_label, model_name):
         return get_model(app_label, model_name)
 
 
-def get_models_compat(app_label):
-    """Get models on multiple Django versions."""
+def get_models_compat(app):
+    """Get models on multiple Django versions for an app."""
     try:
         # django >= 1.7
         from django.apps import apps
-        return apps.get_app_config(app_label).get_models()
+        return apps.get_app_config(app).get_models()
```

Comment:
Good point.
Seems you have tried to address this in 3a0ca9e.
But the main problem still is that it behaves differently for Django 1.7 and before.
For Django 1.7 it should be `app.get_models()` then probably.

I've pushed a new commit, which removes it completely and should fix two more commands: 99ecba5.